exec/java-exec/src/main/codegen/templates/VariableLengthVectors.java

```diff
@@ -119,29 +117,22 @@ public int getVarByteLength(){
   @Override
   public SerializedField getMetadata() {
     return getMetadataBuilder() //
+             .addChild(offsetVector.getMetadata())
              .setValueCount(getAccessor().getValueCount()) //
-             .setVarByteLength(getVarByteLength()) //
+//             .setVarByteLength(getVarByteLength()) //
```

Comment:
We should probably at least leave a comment as to why this was here originally and why it was removed and/or may re-enabled at some point. That or just remove it.